book/src/configuration.md

```diff
@@ -217,6 +217,7 @@ Options for rendering whitespace with visible characters. Use `:set whitespace.r
 |-----|-------------|---------|
 | `render` | Whether to render whitespace. May either be `"all"` or `"none"`, or a table with sub-keys `space`, `tab`, and `newline`. | `"none"` |
 | `characters` | Literal characters to use when rendering whitespace. Sub-keys may be any of `tab`, `space`, `nbsp`, `newline` or `tabpad` | See example below |
+| `clear-whitespace-line-on-ret` | If current line is only whitespace, whether to clear it when `<ret>` is pressed | `true`
```

Comment:
`editor.whitespace` is about rendering whitespace with characters like `·`. This should live at the top-level of `editor` instead